src/schemas/json/pyproject.json

```diff
@@ -160,7 +160,7 @@
         "git": {
           "type": "string",
           "description": "The url of the git repository.",
-          "format": "uri"
+          "pattern": "^([A-Za-z0-9\\-]+@|https://|http://)[A-Za-z0-9\\-]+\\.[A-Za-z0-9\\-]+(:|/)[A-Za-z0-9\\-]+/[A-Za-z0-9\\-]+\\.git$"
```

Comment:
Would it be possible to use a `anyOf` here, one item being the `format` property - and the other item being your new `pattern` to accept Git repository URLs? I worry that there are valid `uri`s that will no longer be accepted by the new `pattern`